src/commands/add-cx-server.ts

```diff
@@ -29,16 +29,26 @@ export default class AddCxServer extends Command {
     const options = await this.getOptions();
 
     try {
-      cli.action.start('Reading templates');
-      const files = readTemplates({ from: [path.resolve(__dirname, '..', 'templates', 'add-cx-server')], to: flags.projectDir });
-      cli.action.stop();
+      const githubPrefix = 'https://raw.githubusercontent.com/SAP/devops-docker-cx-server/master/cx-server-companion/life-cycle-scripts/';
+      const files = [
+        {
+          sourcePath: new URL('cx-server', githubPrefix),
+          targetFolder: path.resolve(flags.projectDir, 'cx-server'),
+          fileName: path.resolve(flags.projectDir, 'cx-server', 'cx-server')
+        },
+        {
+          sourcePath: new URL('server.cfg', githubPrefix),
+          targetFolder: path.resolve(flags.projectDir, 'cx-server'),
+          fileName: path.resolve(flags.projectDir, 'cx-server', 'server.cfg')
+        }
+      ];
```

Comment:
what happened to the `.bat` file? Shouldn't we keep Windows support?

Reply:
Also what happened to the concern about the number of relevant files potentially changing? No proper/sane way to implement it?

Reply:
So this is the result from a discussion with the pipeline team. They said that they consider only those two files essential and also consider the naming and folder structure as stable. So this seemed like the easiest way.

There are downsides, like the risk of breaking this command when the file names change. But I feel any other solution would likely be over-engineered or slow.

Henning also pointed out that we should keep the `.bat` while the pipeline team seemed to dismiss it. I will add it back in if the command is executed on a Windows machine.